src/dotnet/commands/dotnet-complete/ParseCommand.cs

```diff
@@ -24,6 +24,13 @@ public static int Run(string[] args)
 
             Console.WriteLine(result.Diagram());
 
+            var optionValuesToBeForwarded = result.AppliedCommand()
+                                                  .OptionValuesToBeForwarded();
+            if (optionValuesToBeForwarded.Any())
+            {
+                Console.WriteLine("Option values to be forwarded: ");
```

Comment:
We don't need to run the loc tool any more (it's getting automated), but please continue to put all strings in separate localizablestrings.cs so that we don't have to go hunting for them yet again.